concent_api/core/views.py

```diff
@@ -1443,6 +1443,114 @@ def store_message(
     return stored_message
 
 
+def update_concent_states(
+    client_public_key: str,
+):
+    active_subtask_states = [
+        'forcing_report',
+        'forcing_result_transfer',
+        'forcing_acceptance',
+        'additional_verification',
+    ]
+    clients_subtask_list_as_requestor   = Subtask.objects.filter(requestor = client_public_key, delivered = False, state__in = active_subtask_states)
```

Comment:
`requestor` and `provider` fields are relation to Client, so you have to query it like `requestor__public_key=`

Also, you can do this in single query. 
```
from django.models.db import Q
Subtask.objects.filter(
    delivered = False, state__in = active_subtask_states,
    Q(requestor__public_key = client_public_key) | Q(provider__public_key = client_public_key)
)
```

If the part with Q's will be repeated more than once in our code we will add it as manager's method.